content/reference/endpoints/inbox_attachments.md

```diff
@@ -28,22 +33,23 @@ updated_at        | [Time](/reference/enums#formats) | Read       | Attachment's
 {: class="table table-bordered"}
   </div>
   <div class="tab-pane" id="inbox_write" markdown="1">
-Name              | Type    | Read/Write | Description
-------------------|---------|------------|------------
-id                | Integer | Read       | Attachment's id.
-account_id        | Integer | Read       | Id of Account Attachment belongs to.
-------------------|---------|------------|------------
-remote_file_url   | String  | Write      | Attachment's remote url address.
-file              | String  | Write      | Attachment's Base64 encoded file prepended with `data:image/png;base64,` (note comma separator between the type prefix and the Base64-encoded body)
-url               | String  | Read       | Attachment's url address.
-name              | String  | Read/Write | Attachment's name.
-image_width       | Integer | Read       | Attachment's image width in pixels.
-image_height      | Integer | Read       | Attachment's image height in pixels.
-file_size         | Integer | Read       | Attachment's file size in bytes.
-file_content_type | String  | Read       | Attachment's file content type.
-------------------|---------|------------|------------
-created_at        | [Time](/reference/enums#formats) | Read       | Attachment's create time.
-updated_at        | [Time](/reference/enums#formats) | Read       | Attachment's update time.
+Name              | Type    | Read/Write | Description | Constraints
+------------------|---------|------------|-------------| 
+account           | Integer | Read       | ID of Account the Attachment belongs to. |
+messages          | Array   | Read       | IDs of Messages the Attachment belongs to. |
+------------------|---------|------------|------------|
+id                | Integer | Read       | Attachment's id. |
+remote_file_url   | String  | Write      | Attachment's remote url address. | Must be a valid URL - use either this attribute or `file`
+file              | String  | Write      | Attachment's Base64 encoded file prepended with `data:image/png;base64,` (note comma separator between the type prefix and the Base64-encoded body) | Must be less than: 10 megabytes, use either this attribute or `remote_file_url`  
+url               | String  | Read       | Attachment's url address. |
+name              | String  | Read/Write | Attachment's name. |
+image_width       | Integer | Read       | Attachment's image width in pixels. |
+image_height      | Integer | Read       | Attachment's image height in pixels. | 
+file_size         | Integer | Read       | Attachment's file size in bytes. |
+file_content_type | String  | Read       | Attachment's file content type. |
```

Comment:
I think all of them should be `Read/Write` under write scope

Reply:
They are not, check the whitelisted attributes in the controller